src/clusterfuzz/_internal/base/tasks.py

```diff
@@ -448,6 +446,48 @@ def lease(self, _event=None):  # pylint: disable=arguments-differ
     track_task_end()
 
 
+class PostprocessPubSubTask(PubSubTask):
+  """A postprocess task received over pub/sub."""
+
+  def __init__(self,
+               output_url_argument,
+               pubsub_message,
+               is_command_override=False):
+    command = 'postprocess'
+    job_type = 'none'
+    eta = None
+    high_end = False
+    grandparent_class = super(PubSubTask, self)
+    grandparent_class.__init__(command, output_url_argument, job_type, eta,
+                               is_command_override, high_end)
+    self._pubsub_message = pubsub_message
+
+  @contextlib.contextmanager
+  def lease(self, _event=None):  # pylint: disable=arguments-differ
```

Comment:
Are there big differences from the other lease()? can we avoid duplicated code here?